app/api/bread/schema.py

```diff
@@ -0,0 +1,36 @@
+from marshmallow import fields, ValidationError
+
+from app import ma
+from app.models import BreadType
+
+
+def validate_not_none(value):
+    """
+    Validates whether the given value is None.
+    """
+    if value is None:
+        raise ValidationError('Items contains an unknown material type')
+
+
+class OrderSchema(ma.Schema):
+    id = fields.Integer()
+    date = fields.Date()
+    items = fields.Function(
+        lambda order: list(map(lambda item: item.name, order.items))
+    )
+    active = fields.Boolean()
+    editable = fields.Boolean()
```

Comment:
editable is genoeg voor de frontend (zie models.py)

Reply:
Ik ga die toch nog even daar houden, stel dat ik de boodschap "geen brood deze week" wil weergeven ofzo.